envisage/api.py

```diff
@@ -65,6 +65,9 @@
 from .i_plugin_manager import IPluginManager
 from .i_service_registry import IServiceRegistry
 
+from .ids import BINDINGS, COMMANDS, PREFERENCES, PREFERENCES_CATEGORIES, \
```

Comment:
Style note: on this project, we tend to break up long import lines to place the imports one-per-line. For an example, see the `from .extension_point_binding` line below.

I'd also like to keep the imports alphabetically ordered as far as possible - this makes it easier for a code reader to scan the set of imports to see if something's there or not (and also helps to minimise merge conflicts in the event that two separate PRs both touch the import list). Please could you move this addition so that it's just before the `from .import_manager` import?